lib/ansible/modules/crypto/entrust/ecs_certificate.py

```diff
@@ -0,0 +1,890 @@
+                type: str
+            email4:
+                description: Custom email field.
+                type: str
+            email5:
+                description: Custom email field.
+                type: str
+            dropdown1:
+                description: Custom dropdown field.
+                type: str
+            dropdown2:
+                description: Custom dropdown field.
+                type: str
+            dropdown3:
+                description: Custom dropdown field.
+                type: str
+            dropdown4:
+                description: Custom dropdown field.
+                type: str
+            dropdown5:
+                description: Custom dropdown field.
+                type: str
+    cert_expiry:
+        description:
+            - The date the certificate should be set to expire, as an RFC3339 compliant date. For example, C(2020-02-23).
+            - I(cert_expiry) is only supported for requests of C(request_type=NEW) or C(request_type=RENEW). If C(request_type=REISSUE),
+              I(cert_expiry) will be used for the first certificate issuance, but subsequent issuances will have the same expiry as the initial
+              certificate.
+            - A reissued certificate will always have the same expiry as the original certificate.
+            - Note that only the date (day, month, year) is supported for specifying expiry date. If you choose to specify an expiry time with the expiry date,
+              the time will be adjusted to Eastern Standard Time (EST). This could have the unintended effect of moving your expiry date to the previous day.
+            - Applies only to accounts with a pooling inventory model.
+            - Only one of I(cert_expiry) or I(cert_lifetime) may be specified.
+        type: str
+    cert_lifetime:
+        description:
+            - The lifetime of the certificate.
+            - Applies to all certificates for accounts with a non-pooling inventory model.
+            - I(cert_lifetime) is only supported for requests of C(request_type=NEW) or C(request_type=RENEW). If C(request_type=REISSUE), I(cert_lifetime) will
+              be used for the first certificate issuance, but subsequent issuances will have the same expiry as the initial certificate.
+            - Applies to certificates of I(cert_type)=C(CDS_INDIVIDUAL, CDS_GROUP, CDS_ENT_LITE, CDS_ENT_PRO, SMIME_ENT) for accounts with a pooling inventory
+              model.
+            - Only one of I(cert_expiry) or I(cert_lifetime) may be specified.
+        type: str
+        choices: [ P1Y, P2Y, P3Y ]
+seealso:
+    - module: openssl_privatekey
+      description: Can be used to create private keys (both for certificates and accounts).
+    - module: openssl_csr
+      description: Can be used to create a Certificate Signing Request (CSR).
+    - module: certificate_complete_chain
+      description: Allows to find the root certificate for the returned fullchain.
```

Comment:
Now you mention that the returned chain is a fullchain, but doesn't include the root (?). Above, you wrote that `chain_file` also contains the root. Or did I understood something wrong?

Reply:
Nope, I was using other modules as examples for how to format the seealso and didn't update it when I added the chain_file param. This see_also is obsolete given that.